README.md

```diff
@@ -175,13 +177,15 @@ exponentiated_gradient.fit(X, y, sensitive_features=sensitive_features)
 randomized_predictions = exponentiated_gradient.predict(X)
 ```
 
-Please open a [new issue](https://github.com/fairlearn/fairlearn/issues) if you encounter any problems.
+Please open a [new issue](https://github.com/fairlearn/fairlearn/issues) if you
+encounter any problems.
 
 </details>
 
 ## Usage
 
-For common usage refer to the [Jupyter notebooks](./notebooks) and our [API guide](./CONTRIBUTING.md#api)
+For common usage refer to the [Jupyter notebooks](./notebooks) and our [API
+guide](./CONTRIBUTING.md#api)
```

Comment:
Nitpicking, but could you put the whole of the []() things on a single line?

Reply:
There are a few other places where this happens.

Reply:
It depends on what people consider more important, line length or everything in one line. @adrinjalali recommended staying within 79 chars for markdown a while back so I've tried conforming to that when possible. I do see your consideration as relevant as well, @riedgar-ms . I personally don't feel strongly either way

Reply:
Yeah I very much prefer the short lines, but I'm used to the broken lines like these ones here and I understand they may not be easy to read if you're not used to them. I'm fine if you rearrange the lines to have the links together with the caption. You could break before `[API...` here for instance. But at the end of the day, these are not rules written on stone and shouldn't cause any stir or cause developers not to be happy with their codebase. I'm happy if you leave the lines long every now and then if it makes @riedgar-ms happy ;)

Reply:
I think I've addressed all of them